mypy: make check fails with --all-extras due to LiveKitAPI attr-defined

## Summary
`make check` fails when optional extras are installed (`uv sync --frozen --all-extras`).

## Reproduction
```bash
uv sync --frozen --all-extras
make check
```

## Actual
Mypy fails with:
```text
src/voice/sip_setup.py:12: error: Module has no attribute "LiveKitAPI"  [attr-defined]
```

## Expected
`make check` should be green regardless of whether extras are installed.

## Context
Current CI lint job uses `uv sync --frozen` and passes, so this is hidden in default CI profile.

## Proposed fix
- update typing/import strategy in `src/voice/sip_setup.py` for current `livekit` package API
- or add a precise typing guard/override for this symbol
- verify with both profiles:
  - `uv sync --frozen && make check`
  - `uv sync --frozen --all-extras && make check`

--- a/src/voice/sip_setup.py
+++ b/src/voice/sip_setup.py
@@ -2,6 +2,7 @@
 
 import asyncio
 import os
+from typing import Any, cast
 
 from livekit import api
 from livekit.protocol.sip import CreateSIPOutboundTrunkRequest, SIPOutboundTrunkInfo
@@ -9,7 +10,7 @@ from livekit.protocol.sip import CreateSIPOutboundTrunkRequest, SIPOutboundTrunk
 
 async def setup_lifecell_trunk() -> str:
     """Create lifecell outbound SIP trunk. Returns trunk ID."""
-    lk = api.LiveKitAPI(
+    lk = cast(Any, api).LiveKitAPI(
         url=os.getenv("LIVEKIT_URL", "http://localhost:7880"),
         api_key=os.getenv("LIVEKIT_API_KEY", "devkey"),
         api_secret=os.getenv("LIVEKIT_API_SECRET", "secret"),

